ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java

```diff
@@ -15072,13 +15093,13 @@ private boolean queryTypeCanUseCache() {
       return false;
     }
 
-    if (qb.getParseInfo().isAnalyzeCommand()) {
-      return false;
-    }
+      if (qb.getParseInfo().isAnalyzeCommand()) {
```

Comment:
What are these whitespace changes for? Was the nesting bad before?

Reply:
Whitespace earlier wasn't in accordance with checkstyle conventions